try:
	a = 1/0
except Exception as e:
	print("hello")
finally:
	print("final")